src/components/Datatable/Table/Table.tsx

```diff
@@ -199,17 +231,28 @@ TableProps<D>): React.ReactElement => {
     };
   }, [setColumnOrder]);
 
-  useEffect(() => {
-    const unsubscribe = DatatableStore.subscribe(
-      (s) => s.filters,
-      (filters) => {
-        if (any(propEq('isLoading', true))(filters)) gotoPage(0);
-      },
-    );
-    return () => {
-      unsubscribe();
-    };
-  }, [setColumnOrder, gotoPage]);
+  useEffect(
+    () =>
+      DatatableStore.subscribe(
```

Comment:
Any reason why you not unsubscribing?

Reply:
ah I was afraid this might be called out :) so the thing is, it actually does if you look carefully (it returns the `unsubscribe` function returned from the `subscribe` method but it means it's probably not so easy to read, just wanted to save a couple of lines 😄

Reply:
oh I see now, there aren't parenthesis. Yep it's easy to miss

Reply:
Again that's fine if we use it consistently throughout codebase, so either stick to current style or change it everywhere

Reply:
yea will revert :)